src/js/stores/DCOSStore.js

```diff
@@ -204,6 +204,7 @@ class DCOSStore extends EventEmitter {
       );
     });
 
+    this.serviceTree = null;
```

Comment:
Why not have a method that set `serviceTree` to `null`?

Reply:
We already have an `if event is DCOS_CHANGE` in `this.emit` - wouldn't that be a better place for this reset? 🤔

Reply:
I agree here. IMHO it would be better to have a cache cleaning method like `clearServiceTreeCache()`

Reply:
The goal here is to lazily evaluate the tree so if no one calls `get serviceTree` we don't compute the tree at all. At the same time, if any new data comes, we need to clear the cached tree by setting it to null. This improves on the previous behavior because we would always recompute the tree when we have new data.